src/EA.Weee.Web/Areas/Aatf/Mappings/ToViewModel/EditDraftReturnNotesViewModelMap.cs

```diff
@@ -18,8 +18,7 @@ public EditDraftReturnedNotesViewModel Map(EvidenceNotesViewModelTransfer source
         {
             Condition.Requires(source).IsNotNull();
 
-            //TODO: place holder map values until the compliance year is put in for AATF
-            return Map(source.Notes, new DateTime(), null);
+            return Map(source.Notes, SystemTime.UtcNow, null);
```

Comment:
We need to expand the EvidenceNotesViewModelTransfer class to have a DateTime property, called CurrentDate, that should be passed in the constructor.  When doing new EvidenceNotesViewModelTransfer in the controller pass in the currentDate that is retrieved from GetApiUtcDate call. Pass this date into Map rather than SystemTime.UtcNow.

The reason we use GetApiUtcDate is that it allows the testing to put WEEE into a date other than the current date. It uses the FixedCurrentDate value from [dbo].[SystemData].